jdbc/src/main/java/org/apache/metamodel/jdbc/SqlKeywords.java

```diff
@@ -58,6 +58,8 @@
         KEYWORDS.add("DELETE");
         KEYWORDS.add("AND");
         KEYWORDS.add("OR");
+        KEYWORDS.add("END");
```

Comment:
Yes, we can add this too, but what happens if we add all keywords in one shot, is any performance degrade?
Adding columns one by one is not a good approach.
Thoughts?

Thanks,
Laxmilal Menaria


Sent from my iPhone




On Mon, Aug 29, 2016 at 10:07 PM +0530, "Kasper Sørensen" <notifications@github.com> wrote:












In jdbc/src/main/java/org/apache/metamodel/jdbc/SqlKeywords.java:
> @@ -58,6 +58,8 @@
>          KEYWORDS.add("DELETE");
>          KEYWORDS.add("AND");
>          KEYWORDS.add("OR");
> +        KEYWORDS.add("END");



If we add "END" then we should probably also add "BEGIN" :)



—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.